packages/grafana-data/src/types/datasource.ts

```diff
@@ -307,8 +298,6 @@ abstract class DataSourceApi<
 
   getVersion?(optionalOptions?: any): Promise<string>;
 
-  showContextToggle?(row?: LogRowModel): boolean;
```

Comment:
This is part of the public api, not sure we can just remove it like this. I think we need to deprecate it first.

Reply:
it is, moved up just ^

Reply:
hehehe it was removed in a previous commit and then added back 😄

Reply:
Yep, sorry for going back and forth. We will keep both until 9.0  🚀  #39823